tf2_kdl/CMakeLists.txt

```diff
@@ -1,44 +1,54 @@
 cmake_minimum_required(VERSION 3.5)
 project(tf2_kdl)
 
-find_package(orocos_kdl)
-find_package(catkin REQUIRED COMPONENTS cmake_modules tf2 tf2_ros tf2_msgs)
-
-find_package(Eigen REQUIRED)
-
-catkin_package(
-  INCLUDE_DIRS include ${Eigen_INCLUDE_DIRS}
-  DEPENDS Eigen orocos_kdl
-)
+if(NOT WIN32)
+  add_definitions(-std=c++11)
```

Comment:
Why c++11? Why not 14. Add a note explaining this so that future devs know when it can be removed

Reply:
Thanks @mlautman 28d444e12eab3cc0c1be879426fbfa8f8cca48b2